clang/File.d

```diff
@@ -13,8 +13,13 @@ struct File
 {
     mixin CX;
 
-    @property string name ()
+    string name() const
     {
-        return toD(clang_getFileName(cx));
+        return toD(clang_getFileName(cast(void*) cx));
```

Comment:
I suggest removing it for now. I think it's better to not use const at all than using const and breaking it. Note that const in D has much stronger guarantees that in C++.